runtime/src/cost_tracker.rs

```diff
@@ -144,6 +148,22 @@ impl CostTracker {
         Ok(())
     }
 
+    // evaludate transaction's aggregated instruction executino cost against tx-wide cap, 
+    // throw it out up front if exceeds the cap, since it is *likely* to be dropped when executed
+    // down the road. 
+    // *likely* because instruction actual cost wouldn't be known until it is executed, and the
+    // accounts status. Cost model use replayed per-program measure to estimate instruction cost, 
+    // We could put a factor of 1.25 on tx-wide cap to reduce false negative.
+    fn satisfy_transaction_units_cap(&self, tx_cost: &TransactionCost) -> Result<(), CostTrackerError> {
+        let tx_wide_cap = 200_000u64;
```

Comment:
Will have to look at the tx itself to determine if the there exists a compute program ix requesting a higher cap.  Also, there will initially be some exceptions that need to be taken into account as well: https://github.com/solana-labs/solana/pull/21743

Reply:
so the default is 200_000, max is 1_400_000. `compute_budget::process_transaction(&tx)` will give me the requested cap, or error. cost_tracker will use that to filter txs

Reply:
Yeah, though not great having process that twice, be nice to optimize that so it's done once and then maybe passed in with the batch

Reply:
Makes great sense, will do just that